app/views/user_plans/index.html.erb

```diff
@@ -0,0 +1,177 @@
+<main>
+  <%= render "partials/sidebar_dashboard" %>
+  <section class="main container center-align pricing-plans">
+    <div class="row">
+      <section class="col s12">
+        <div class="row">
+          <h2 class="header">User Plans</h2>
+          <% if current_user.user_type == "taskee" %>
```

Comment:
These plan cards could be a partial which could be shared by all the plans. Which means both the plan title, price, plan benefits(receive 10 tasks, earn reputation etc) should come from the database.